x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -215,8 +216,16 @@ private static Failure fail(Node<?> source, String message, Object... args) {
         // if there are no (major) unresolved failures, do more in-depth analysis
 
         if (failures.isEmpty()) {
+            Set<Failure> localFailures = new LinkedHashSet<>();
             final Map<Attribute, Expression> collectRefs = new LinkedHashMap<>();
 
+            // Full-Text search function are not allowed in the SELECT clause
+            plan.forEachUp(p -> p.forEachExpressionsUp(e -> {
+                if (e instanceof FullTextPredicate) {
+                    localFailures.add(fail(e, "Cannot use a Full-Text search functions in the SELECT clause"));
```

Comment:
The error message I think has several issues: why the capital letters for Full-Text, and why the plural `functions`?Also, if you look at our docs - https://www.elastic.co/guide/en/elasticsearch/reference/7.x/sql-functions-search.html - full-text search functions include `SCORE()` as well, but Score is not an `instanceof FullTextPredicate` so it wouldn't match this condition.

So, try to adjust the error message not to partially conflict with the documentation.

Reply:
Thx, Rephrased the error message.
I chose not to refer to the exact usage in the user's query because it can be a large string, given the params that would be often used, e.g.:
```
QUERY('Man*', 'default_field=last_name;lenient=true', 'fuzzy_rewrite=scoring_boolean')
```